package.json

```diff
@@ -14,51 +16,64 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "@embroider/util": "^0.39.1 || ^0.40.0 || ^0.41.0",
-    "ember-cli-babel": "^7.17.2",
-    "ember-cli-htmlbars": "^5.1.0"
+    "@embroider/util": ">= 0.39.1 < 2.0.0-alpha.1",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.0.1"
   },
   "devDependencies": {
-    "@ember/optional-features": "^1.3.0",
-    "@embroider/macros": "^0.39.1 || ^0.40.0 || ^0.41.0",
-    "@embroider/test-setup": "^0.39.1 || ^0.40.0 || ^0.41.0",
-    "@glimmer/component": "^1.0.0",
-    "@glimmer/tracking": "^1.0.0",
-    "babel-eslint": "^10.0.3",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.6.0",
+    "@embroider/macros": "^0.50.2",
+    "@embroider/test-setup": "^0.50.2",
```

Comment:
Let's upgrade all embroider deps to `^1.0.0`?

This is already part of the 4.1 addon blueprint, probably PR was created using ember-cli-udate for version 4.0?